src/integrationTest/java/org/opensearch/security/DoNotFailOnForbiddenTests.java

```diff
@@ -443,6 +444,23 @@ public void shouldPerformCatIndices_positive() throws IOException {
         }
     }
 
+    @Test
+    public void shouldPerformCatAliases_positive() throws IOException {
+        try (RestHighLevelClient restHighLevelClient = cluster.getRestHighLevelClient(LIMITED_USER)) {
```

Comment:
wdyt about performing the same request as the admin user and verify that the response contains `horrible_songs`?

Reply:
Added a test here: [d0b0882](https://github.com/opensearch-project/security/pull/4436/commits/d0b0882ec271d67d651f671640faf52bc1600e52)